tests/components/switchbee/common.py

```diff
@@ -0,0 +1,37 @@
+"""Common methods used across the tests for switchbee devices."""
```

Comment:
Adding tests is great, but maybe keep those for a follow-up PR to reduce the size of the PR

Reply:
> Adding tests is great, but maybe keep those for a follow-up PR to reduce the size of the PR

Ok, I'll remove them from this PR for now